server/libs/infra/src/db/migrations/1679662469076-RemoveChecksumIndex.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class removeChecksumIndex1679662469076 implements MigrationInterface {
+  name = 'removeChecksumIndex1679662469076';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`DROP INDEX "public"."IDX_64c507300988dd1764f9a6530c"`);
+    await queryRunner.query(`ALTER TABLE "assets" DROP CONSTRAINT "UQ_userid_checksum"`);
```

Comment:
I'm guessing you manually removed the `ADD CONSTRAINT "UQ_userid_checksum"`? You should also remove this query, otherwise you're only removing the constraint here and not adding it back later.

Reply:
We probably need to regenerate the migrations for this PR as they became such a mess. I might need some help on that when we approach merge tiem

Reply:
I've updated the migrations